### PR TITLE
Fixes #2 - Welcome Modal

### LINKED ISSUE #2: Add Welcome Screen
- Detect when someone is a first time user based on their local storage.
- If a user is a first time user, show them a welcome screen (possibly a modal) that explains how to use the app.

### PR DESCRIPTION
**Pull request by mftruso**
- Updates to Bootstrap 5
- Shows a welcome modal with basic instructions the first time a user visits the site on their device.

### CODE REVIEW

File: src/WelcomeModal.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <>
+      <Modal show={show} onHide={handleClose} size="xl">
+        <Modal.Header closeButton>
+          <Modal.Title>Welcome!</Modal.Title>
+        </Modal.Header>
+        <Modal.Body><span className="text-primary">ConfessIt</span> is a tool to help Roman Catholics walk through an examination of conscience prior to going to confession. 
+        We hope you'll find this useful to help remember sins you've committed since your last confession. 
+        Just check the <strong>Yes</strong> box next to sins in the Examine list, or tap the plus button to add your own.
```
Comment (kas-catholic) [inline src/WelcomeModal.js:26]: A bit nit-picky, but let's make `Examine` strong too since it's the name of a column, like Review and Walkthrough.

**Review by kas-catholic — APPROVED** [review]
Love it! And you upgraded Bootstrap :heart_eyes: 

Might need a rebase to resolve conflicts in `yarn.lock`.

File: src/WelcomeModal.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+  },[])
+
+  return (
+    <>
+      <Modal show={show} onHide={handleClose} size="xl">
+        <Modal.Header closeButton>
+          <Modal.Title>Welcome!</Modal.Title>
+        </Modal.Header>
+        <Modal.Body><span className="text-primary">ConfessIt</span> is a tool to help Roman Catholics walk through an examination of conscience prior to going to confession.
```
Comment (kas-catholic) [inline src/WelcomeModal.js:24]: Can we use a `Trans` component here to make this a translated string? (English only is fine for now, but putting it in a component will let us add translations for it later. See `About.js` for an example:

https://github.com/kas-catholic/confessit-web/blob/0f6f205b9b86dd1716d68769fc1e5e3fbb726ae3/src/About.js#L16-L18

File: src/WelcomeModal.js (hunk 1 above)
Comment (mftruso) [reply src/WelcomeModal.js]: fixed

File: src/WelcomeModal.js (hunk 2 above)
Comment (mftruso) [reply src/WelcomeModal.js]: added!

**Comment by mftruso** [comment]
a lint error [broke](https://app.netlify.com/sites/confessit-web/deploys/634d9ff935ca5400083e410e) the netlify deploy, so i added it as a CI step

**Review by mkasberg — APPROVED** [review]

**Pull request merged**